### ISSUE TITLE
[ENH] Be able to generate prediction intervals for exponential smoothing models

### ISSUE THREAD

**Issue by GianlucaGuarroSonos** [sktime/sktime#4072]
**Is your feature request related to a problem? Please describe.**
When you try to run the function `predict_intervals` off an ExponentialSmoothing forecasting object, you get the following error:
"NotImplementedError: ExponentialSmoothing does not have the capability to return prediction intervals. If you think this estimator should have the capability, please open an issue on sktime."

**Describe the solution you'd like**
It seems like statsmodels has two generalized exponential smoothing classes. By generalized I mean that they can contain trends and seasonalities. These are [ETSModel](https://www.statsmodels.org/dev/generated/statsmodels.tsa.exponential_smoothing.ets.ETSModel.html) and [holtswinter.ExponentialSmoothing](https://www.statsmodels.org/dev/generated/statsmodels.tsa.holtwinters.ExponentialSmoothing.html). Sktime currently interfaces holtswinter.ExponentialSmoothing but I think it should try to interface ETSModel instead as it is the more feature-rich version.

As you can see at the end of [this page](https://www.statsmodels.org/dev/examples/notebooks/generated/ets.html), ETSModel can do prediction intervals and simulations.

**Additional context**
This feature request is similar to https://github.com/sktime/sktime/issues/1362

**Comment by fkiraly** [comment]
Nice idea! Sounds like a good first issue to me.

Tutorial on implementing or interfacing estimators is here:
https://www.sktime.org/en/stable/developer_guide/add_estimators.html
contains a link to a video tutorial.

The learning task is "forecasting", and the reqest implies that `predict_interval` and/or `predict_quantiles` should be implemented, although that could be done in a second PR.

**Comment by fkiraly** [comment]
Actually, I think this is already done.

The model `AutoETS` is a direct interface to `ETSModel` under default parameter settings. It only does automated model selection when the parameter `auto=True` (but default is `False` which is vanilla `ETSModel` behaviour).

Does this satisfy your requirements, @GianlucaGuarroSonos?

**Comment by fkiraly** [comment]
I improved the docstring of `AutoETS` so it should be clearer now:
https://github.com/sktime/sktime/pull/4116

**Comment by GianlucaGuarroSonos** [comment]
Thanks @fkiraly for looking into this. I'll have to try using AutoETS then and see how it works out for me. 

I will say that the naming is a little confusing. Wouldn't it make more sense for the structure to be similar to what sktime has for AutoARIMA and ARIMA? That is, why not break up AutoETS into AutoETS and ETS?

**Comment by fkiraly** [comment]
> I will say that the naming is a little confusing. Wouldn't it make more sense for the structure to be similar to what sktime has for AutoARIMA and ARIMA? That is, why not break up AutoETS into AutoETS and ETS?

I actually agree with you after careful thinking.

What I first thought is that the interface does not exist, based on the confusing naming convention. That's why I first categorized this "good first issue" and then thought, wait a moment.

This shows that the status quo is a bit confusing for users.

> Btw, we can close this but thought I would bring this up. Maybe there is a good reason to have the logic for AutoETS and "manual" ETS together?

How about we split it up?
Keep `AutoETS` as-is, and a separate `ETS` class or similar. Later we could think about whether `AutoETS` should internally rely on `ETS`, by inheritance or composition.

An alternative would be exposing `ETS` delegating to `AutoETS` with fixed parameters (via `_DelegatedForecaster`). That miht be a minimal amount of lines to implement.

What do you think?

**Comment by GianlucaGuarroSonos** [comment]
Yeah I think splitting it up makes sense. And I agree that we can hold off on deciding how exactly AutoETS should rely on ETS. For me at least, I would need to understand the codebase better to make an informed decision on that. 

> An alternative would be exposing ETS delegating to AutoETS with fixed parameters (via _DelegatedForecaster). That miht be a minimal amount of lines to implement.

This bit I could not fully understand as I do not know what `_DelegatedForecaster` and its purpose

**Comment by fkiraly** [comment]
> This bit I could not fully understand as I do not know what `_DelegatedForecaster` and its purpose

It's a mixin which patches through methods, and which could also be used to expose another estimator with specific parameters set, as a new estimator. E.g., `AutoETS` (with `auto=False`) as `ETS`.

**Comment by kumar781** [comment]
Is there a way to set AutoEts(Auto= True ,seasonal = 'add', trend= 'add' ,damped_trend=True, bound = {'damping_bound':(0,0.4)})
or something like AutoEts(Auto= False ,seasonal = None, trend= 'add' ,damped_trend=True, bound = {'damping_bound':(0,0.4)}) ?
I want this setting to counter the unusual seasonality in my auto ets forecasts
